source/write_docs/add_openapi_spec/index.html.md.erb

```diff
@@ -1,48 +1,48 @@
 ---
-title: Add an OpenAPI specification
+title: Convert an OpenAPI specification into documentation
 weight: 42
-last_reviewed_on: 2020-07-01
+last_reviewed_on: 2021-07-06
 review_in: 1 year
 ---
 
-# Add an OpenAPI specification to your documentation
+# Convert an OpenAPI specification into documentation
 
-OpenAPI is an API description format for RESTful APIs. The GDS Open Standards board [recommends that government organisations use OpenAPI version 3 to describe RESTful APIs][use-openapi].
+OpenAPI is an API description format for REST APIs. Government's Open Standards board [recommends using OpenAPI][use-openapi].
 
-The template can extract endpoint information from an [Open API v3 specification][openapi3-spec] in a YAML file.
+You can convert an OpenAPI file into an API reference page in your documentation.
 
-## Amend the tech-docs.yml file
+Your OpenAPI file must be both:
 
-Add the following to your tech-docs.yml file:
+- [version 3](https://github.com/OAI/OpenAPI-Specification/blob/main/versions/3.0.0.md)
+- in YAML format
 
-```yaml
-api_path: path/to/file.yaml
-```
+1. Add the following to your `config/tech-docs.yml` file:
 
-This can be a relative path to a file in your tech docs repo folder, for example:
+    ```yaml
+    api_path: <OPENAPI_FILE>
+    ```
 
-```yaml
-api_path: ./source/pets.yaml
-```
+    where `<OPENAPI_FILE>` is your OpenAPI file, for example `/openapi/openapi.yaml`.
 
-This can also be a URL to a file hosted elsewhere, for example:
+2. Add the following in your documentation to convert the OpenAPI file into an API reference:
 
-```yaml
-api_path: https://raw.githubusercontent.com/OAI/OpenAPI-Specification/master/examples/v3.0/petstore.yaml
-```
+    ```markdown
+    api>
```

Comment:
Is there anything missing here? Line 33 refers to `api_path` but i don't see `api_path` in this code

Reply:
Ah sorry, line 33 refers to step 1 not step 2. But I can't slot line 33's content into step 1 because it would create sub-bullets.

Reply:
ah i understand, thank you